acsoo/tag_requirements.py

```diff
@@ -44,15 +44,15 @@ def _get_last_sha(filename):
     # find the short sha of the last commit to filename
     return check_output([
         'git', 'log', '-n', '1', '--format=%h', filename
```

Comment:
I wasn't sure if I could use this argument because based on what I read, it opens the file/stream in text mode, and as this argument already exists in python2.7, I wanted to be sure to not break something else.

Reply:
No risk here: there is no no input, and we want the output as text. And it's better to let the system decide the encoding than hard-coding utf8